SIS_import/sis_upload.rb

```diff
@@ -562,9 +636,8 @@ def create_instructor_sandbox_site(zip_file_name, outputFile)
 	FileUtils.mv(Dir.glob("#{$currentDirectory}*MD5.txt"), $archiveDirectory)
 
 	upload_success_msg = "SIS upload finished with " + fileName
-	outputFile.write(upload_success_msg)
-	p upload_success_msg
+	$logger.info upload_success_msg
 end
 
-# close output file
-outputFile.close unless outputFile == false
+# close logger
+$logger.close
```

Comment:
Will it matter to other code that the global logger is now closed?  Does any other code depend on it being open?